quickjs.c

```diff
@@ -374,7 +374,7 @@ typedef struct JSVarRef {
         struct {
             int __gc_ref_count; /* corresponds to header.ref_count */
             uint8_t __gc_mark; /* corresponds to header.mark/gc_obj_type */
-            bool is_detached;
+            uint8_t is_detached;
```

Comment:
Ah... do IIUC that sizeof(bool) > sizeof(char) on ppc?

Reply:
Bool is 4 bytes in ppc ABI.